packages/editor-ui/src/components/mixins/showMessage.ts

```diff
@@ -17,6 +18,8 @@ export const showMessage = mixins(externalHooks).extend({
 			track = true,
 		) {
 			messageData.dangerouslyUseHTMLString = true;
```

Comment:
There are 76 total occurrences where we use  `showMessage`, `confirmMessage`, or `confirmModal` in our codebase. So it would be a lot of effort to assess each one of them if we need the `dangerouslyUseHTMLString` or not. And IMHO if we do strict stripping of dangerous HTML and enable only a subset of tags and attributes, it's safe to keep it set to `true` because nothing dangerous would get through anyway

Reply:
Fair.. Will create a follow up so we can address this at some point but not yet